Source/DafnyCore/Options/BoogieOptionBag.cs

```diff
@@ -46,6 +46,8 @@ public static class BoogieOptionBag {
 
   public static readonly Option<bool> NoVerify = new("--no-verify", "Skip verification");
 
+  public static readonly Option<bool> DoBoogieTranslation = new("--do-boogie-translation", "Requires --no-verify. Does translation to Boogie but Boogie won't verify it.");
```

Comment:
It seems strange to have an option that can only be used if another option is also used. Why not have the first option imply the second then? Instead of `--do-boogie-translation` why not have `--no-verify-boogie` that tells Dafny to translate to Boogie but not verify it.

That way, whenever the user would type `--no-verify --do-boogie-translation` they only have to type `--no-verify-boogie`. Secondly, I would make this option hidden since it only seems useful for the purpose of debugging the Dafny verification pipeline.

Reply:
Let's do the simple way. Since we already have --bprint, we can use it as the flag to do boogie translation, period. I like that idea.

Reply:
Same for `--sprint`

Reply:
And `--pprint` as well. Added